downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -58,6 +58,17 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 ### New features
 
+#### `data.jsondata` packages
+
+- Introduced constraint annotation validation support.
+- Introduced support for union as expected type.
+
+#### `data.xmldata` packages
+
+- Introduced constraint annotation validation support. 
+- Introduced support for default values in the expected record types.
```

Comment:
Let's change to address comments similar to jsondata above.

Reply:
Updated https://github.com/ballerina-platform/ballerina-dev-website/pull/9202/commits/a3050f3c3ad22f76dcd2d8732cb538ccb3a14100